Remove code signature in macOS release assets (#280)
hexatomic/hexatomic: docs/user/src/installation.md

    - **Windows**: Download `hexatomic-<version>-win32.win32.x86_64.zip`
3. Extract the downloaded `zip` file to a directory of your choice.[^1]

> For Linux and Windows That's it. You can now run Hexatomic.

Reviewer: ```suggestion
For Linux and Windows, that's it. You can now run Hexatomic.
```
Fix typos.

> On macOS, you need to move the Hexatomic application from the "Download" folder to the "Application" folder by dragging it to "Applications" in the Finder.

Reviewer: ```suggestion
On macOS, you need to move the Hexatomic application from the "Download" folder to the "Applications" folder by dragging it to "Applications" in the Finder.
```
Wouldn't the folder also be called "Application**s**"=

- In **Windows**, the launcher file is simply called `hexatomic.exe` and can be started with `hexatomic`.  
- For **Mac OS X**, we provide an `.app` file called `hexatomic.app`.

### macOS-specific instructions

> When you start Hexatomic for the first time on macOS, use the context menu (accessible with a click to both mouse buttons) and choose "Open" from there.

Reviewer: ```suggestion
When you start Hexatomic for the first time on macOS, use the context menu and choose "Open" from there.
```
I think Mac users should be aware of how to open the context menu, and there are more than one ways to do this as well. I'd just drop this here.

![Open Hexatomic from the context menu on macOS](macos-open.png)

> You will be asked if you want to open Hexatomic, even when the developer is unknown.

Reviewer: Doesn't quite reflect what the dialog in the image says. I'd change this as below.

```suggestion
You will be asked if you want to open Hexatomic, although it has not been downloaded from a trusted source.
This dialog appears because we don't have an expensive Apple Developer ID and therefore cannot sign Hexatomic releases with it.
```

Reviewer: If you find it too snarky, feel free to drop "expensive" or replace with "paid" or something...
